arches/app/media/js/viewmodels/card.js

```diff
@@ -328,6 +330,20 @@ define([
                     loading: loading,
                     cardwidgets: params.cardwidgets,
                 });
+            },
+            isFuncNode: function() {
+                var appFuncDesc, appFuncName, nodegroupId;
+                if(params.appliedFunctions && params.card) {
+                    appFuncDesc = self.appliedFunctions[0]['config']['description']['nodegroup_id'];
```

Comment:
@whatisgalen this will result in errors if any of the properties referenced (`self.appliedFunctions[0]['config']['description']['nodegroup_id']`) are undefined. as such, we should probably first check for their presence in the conditional at line 336. Also, this always just checks the first function in the array.  We should probably iterate over `self.appliedFunctions` instead.